build/cluster.json

```diff
@@ -1,54 +1,3 @@
 {
-  "111": 1,
-  "112": 1,
-  "113": 1,
-  "121": 2,
-  "122": 2,
-  "123": 2,
-  "124": 2,
-  "131": 3,
-  "132": 3,
-  "133": 3,
-  "134": 3,
-  "135": 3,
-  "136": 3,
-  "137": 3,
-  "138": 3,
-  "144": 19,
-  "145": 19,
-  "151": 5,
-  "152": 5,
-  "153": 5,
-  "154": 5,
-  "155": 5,
-  "156": 5,
-  "161": 6,
-  "171": 7,
-  "172": 7,
-  "181": 8,
-  "182": 8,
-  "183": 8,
-  "184": 8,
-  "185": 8,
-  "186": 8,
-  "187": 8,
-  "188": 8,
-  "191": 9,
-  "192": 9,
-  "193": 9,
-  "201": 10,
-  "202": 10,
-  "204": 10,
-  "213": 11,
-  "214": 11,
-  "223": 18,
-  "224": 12,
-  "225": 17,
-  "227": 20,
-  "231": 13,
-  "232": 25,
-  "241": 14,
-  "242": 14,
-  "251": 15,
-  "261": 16
+  "121": 2
```

Comment:
valve removed the mappings from this file.  we need to disable this from being updated in the update script